grafana/environments/scaleway-parca-demo/main.jsonnet

```diff
@@ -0,0 +1,44 @@
+local g = import 'grafana.libsonnet';
+
+local grafana = g({
+  namespace: 'parca',
+  datasources+: [
+    {
+      access: 'proxy',
+      editable: false,
+      name: 'Parca - PolarSignals',
+      orgId: 1,
+      type: 'parca-datasource',
+      version: 1,
+      jsonData: { APIEndpoint: 'https://demo.parca.dev' },
+    },
+    {
+      name: 'Parca - Grafana',
+      type: 'parca',
+      url: 'https://demo.parca.dev',
+      editable: false,
+    },
+  ],
+  ingress+: {
+    class: 'nginx',
+    hosts: ['demo.parca.dev'],
```

Comment:
There needs to be a path here too. We only expose Grafana on https://demo.parca.dev/grafana/
Is it possible to somehow add this?

Reply:
`path: '/grafana'` is in our "defaults" under `lib/` for now